docs/docs/distributors/self-serve/run-a-ddex-server.mdx

```diff
@@ -8,34 +8,127 @@ description: Audius Protocol Documentation
 
 import useBaseUrl from '@docusaurus/useBaseUrl'
 
-:::info Coming Soon!
+## Quick Start
 
-Under construction. Please use as your own risk! 🚧
+This guide provides an overview of installing and running a production DDEX server.
 
-Check back soon for updates on how to run and operate own DDEX Ingestion Server.
+> See the full source code and additional documentation for configuring development and production
+> servers on GitHub at [Audius DDEX Processor](https://github.com/AudiusProject/ddex-processor)
 
-:::
-
-## Instructions
+### Provision a Machine
 
-1. Provision a new machine following. We recommend:
+Provision a new machine following. We recommend:
 
 - Ubuntu
 - 8 CPUs
 - 16 to 32GB Memory
 
-2. Visit [audius-docker-compose](https://github.com/AudiusProject/audius-docker-compose) and run the
-   install script to set up your Audius environment on your provisioned machine.
+### Installation
+
+- Clone the [Github Repo](https://github.com/AudiusProject/ddex-processor) with
+  ```bash
+  git clone https://github.com/AudiusProject/ddex-processor.git`
+  ```
+- Install [nodejs](https://deb.nodesource.com/)
+- Install dependencies
+
+  ```bash
+   npm i
+  ```
+
+- Setup a
+  [Cloudflare Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-local-tunnel/)
+
+---
+
+## Configuration
+
+### Configure Cloudflare Tunnel
```

Comment:
is the cloudflare stuff necessary to get off the ground? curious why we added? i would imagine that everyone has a DNS who would want to run this, but also seems like an annoying first step if you don't actually own a domain somewhere you want to do this on